code/modules/clothing/masks/gas_filter.dm

```diff
@@ -39,7 +39,8 @@
 		/datum/gas/nitrium,
 		/datum/gas/freon,
 		/datum/gas/hypernoblium,
-		/datum/gas/bz
+		/datum/gas/bz,
+		/datum/gas/miasma
```

Comment:
Trailing comma? 🤔

Reply:
not sure what you mean

Reply:
```suggestion
		/datum/gas/miasma,
```
they mean this

Reply:
Is that required now per guidelines? Time to re-read the guidelines (last read 2019)

Reply:
it helps people in the future avoid the error you made in the first commit, so we do it where possible

Reply:
The funny error I made in the first commit is me accidentally adding a ``.`` instead of a ``,``

Reply:
yes, but if it had a trailing comma you wouldn't have to add a `,` in the first place